lib/plugins/aws/deploy/compile/events/apiGateway/lib/authorizers.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+const _ = require('lodash');
+
+module.exports = {
+  compileAuthorizers() {
+    _.forEach(this.serverless.service.functions, (functionObject, functionName) => {
+      functionObject.events.forEach(event => {
```

Comment:
Do we need a check if `events` for the function are present (like it was done in https://github.com/serverless/serverless/pull/1599)?